### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by umangyadav**
Remove extraneous `-`.

### CODE REVIEW

**Review by krzysz00 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ To build the static library that is used by MIGraphX
 ```sh
 mkdir build
 cd build
-cmake -G Ninja .. -DBUILD_FAT_LIBROCKCOMPILER=On -DCMAKE_BUILD_TYPE=Release -
+cmake -G Ninja .. -DBUILD_FAT_LIBROCKCOMPILER=On -DCMAKE_BUILD_TYPE=Release
```
Comment (manupak) [inline README.md]: While you are here, can I ask to add `-DCMAKE_C_COMPILER=/opt/rocm/llvm/bin/clang -DCMAKE_CXX_COMPILER=/opt/rocm/llvm/bin/clang++` to be consistent as well.

Comment (umangyadav) [reply README.md]: Done.

**Review by manupak — APPROVED** [review]

**Comment by umangyadav** [comment]
@jerryyin Can this go in ?

**Comment by manupak** [comment]
Thanks @umangyadav!

**Pull request merged**